src/js/Properties.js

```diff
@@ -243,7 +243,7 @@ axs.properties.findTextAlternatives = function(node, textAlternatives, opt_recur
 
     // 1. Skip hidden elements unless the author specifies to use them via an aria-labelledby or
     // aria-describedby being used in the current computation.
-    if (!recursive && !opt_force && axs.utils.isElementOrAncestorHidden(element))
+    if (!opt_force && axs.utils.isElementOrAncestorHidden(element))
```

Comment:
How does this work? opt_force isn't passed in from the getTextFromAriaLabelledby method: https://github.com/GoogleChrome/accessibility-developer-tools/blob/48a69f03f644f9f83b3370e4a72d07b6d704bb1e/src/js/Properties.js#L475